src/utilities/focus.ts

```diff
@@ -130,6 +130,7 @@ export function isElementTabbable(element: HTMLElement): boolean {
     (element.tagName === 'A' ||
       (element.tagName === 'BUTTON' && !(element as HTMLButtonElement).disabled) ||
```

Comment:
Not specific to your change, but why do we need to cast all these as button? @dzearing